homeassistant/components/auth/login_flow.py

```diff
@@ -154,14 +154,18 @@ def __init__(self, flow_mgr, store_result):
     async def _async_flow_result_to_response(self, request, client_id, result):
         """Convert the flow result to a response."""
         if result["type"] != data_entry_flow.RESULT_TYPE_CREATE_ENTRY:
-            if result["type"] == data_entry_flow.RESULT_TYPE_FORM:
-                # @log_invalid_auth does not work here since it returns HTTP 200
-                # need manually log failed login attempts
-                if result.get("errors", {}).get("base") in (
+            # @log_invalid_auth does not work here since it returns HTTP 200
+            # need manually log failed login attempts
```

Comment:
```suggestion
            # @log_invalid_auth does not work here since it returns HTTP 200.
            # We need to manually log failed login attempts.
```